library/core/src/num/int_macros.rs

```diff
@@ -2101,6 +2101,7 @@ macro_rules! int_impl {
         ///
         /// ```
         #[doc = concat!("assert_eq!(100", stringify!($SelfT), ".wrapping_neg(), -100);")]
+        #[doc = concat!("assert_eq!(-100", stringify!($SelfT), ".wrapping_neg(), 100);")]
```

Comment:
```suggestion
        #[doc = concat!("assert_eq!((-100", stringify!($SelfT), ").wrapping_neg(), 100);")]
```
Should there be parentheses, like in the doctest for [`saturating_neg`](https://doc.rust-lang.org/nightly/std/primitive.i32.html#method.saturating_neg)? Otherwise the emitted doctest will have like `-100i32.wrapping_neg()`, which is `-(100i32.wrapping_neg())`, not `(-100i32).wrapping_neg()`. The result is the same, but the former isn't actually calling `wrapping_neg` on a negative number.

Reply:
Oof, good catch! It's already part of a tested rollup so I'll send a follow-up.

Reply:
Added it. :)